src/FileGetContents.php

```diff
@@ -198,6 +266,8 @@ protected function getTlsStreamContextDefaults($cafile)
      * @throws \RuntimeException If https proxy required and OpenSSL uninstalled
      *
      * @return resource Default context
+     *
+     * @final since 1.1.0
      */
     protected function getMergedStreamContext($url)
```

Comment:
I really don't feel comfortable enough with that part to really assess if there's any problematic change that occurred right now so I'll leave it there for now. Might check again later